### PR TITLE
1.10.2 Patch release

### PR DESCRIPTION
**Pull request by ewels**
`dev` > `master` for the 1.10.2 patch release.

3rd release in 24 hours.. When I said that we should release small and often, this isn't really what I meant 😆

### CODE REVIEW

File: .github/workflows/branch.yml [hunk 1]
```diff
@@ -24,9 +24,7 @@ jobs:
           message: |
             Hi @${{ github.event.pull_request.user.login }},
 
-            It looks like this pull-request is has been made against the ${{github.event.pull_request.head.repo.full_name}} `master` branch.
-            The `master` branch on nf-core repositories should always contain code from the latest release.
-            Beacuse of this, PRs to `master` are only allowed if they come from the ${{github.event.pull_request.head.repo.full_name}} `dev` branch.
+            It looks like this pull-request is has been made against the ${{github.event.pull_request.head.repo.full_name}} `master` branch. The `master` branch on nf-core repositories should always contain code from the latest release. Beacuse of this, PRs to `master` are only allowed if they come from the ${{github.event.pull_request.head.repo.full_name}} `dev` branch.
```
Comment (matq007) [inline .github/workflows/branch.yml:27]: It looks like this pull-request ~is~ has been made against the

Comment (ewels) [reply .github/workflows/branch.yml:27]:
```suggestion
            It looks like this pull-request has been made against the ${{github.event.pull_request.head.repo.full_name}} `master` branch. The `master` branch on nf-core repositories should always contain code from the latest release. Beacuse of this, PRs to `master` are only allowed if they come from the ${{github.event.pull_request.head.repo.full_name}} `dev` branch.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,32 @@
 # nf-core/tools: Changelog
 
-## [v1.10.1](https://github.com/nf-core/tools/releases/tag/1.10.1) - [2020-07-30]
+## [v1.10.2 - Copper Camel _(bought back from the dead)_](https://github.com/nf-core/tools/releases/tag/1.10.2) - [2020-07-31]
```
Comment (matq007) [inline CHANGELOG.md:3]:
```suggestion
## [v1.10.2 - Copper Camel _(brought back from the dead)_](https://github.com/nf-core/tools/releases/tag/1.10.2) - [2020-07-31]
```

Comment (ewels) [reply CHANGELOG.md]: 💸 💀

**Pull request merged**